### PR TITLE
feat: Add jwt service

### PR DESCRIPTION
**Pull request by 0sinf**
## What does this PR do?

After google login, create JWT using user information.

## Type of change

- [x] New feature
- [ ] Bug fix
- [ ] Code improvements
- [ ] Breaking change
- [ ] Documentation
- [ ] etc.

## Changes

- Add JWT Module.
- In AuthController, return JWT.

### CODE REVIEW

File: src/jwt/jwt.module.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Module } from '@nestjs/common';
+import { JwtModule } from '@nestjs/jwt';
+import { JwtAuthService } from './jwt.service';
+import { JwtStrategy } from './jwt.strategy';
+
+@Module({
+  imports: [
+    JwtModule.registerAsync({
+      useFactory: async () => ({
```
Comment (marshallku) [inline src/jwt/jwt.module.ts]: Should it be an async function?

Comment (young-sinf) [reply src/jwt/jwt.module.ts:9]: When I tried to use `ConfigModule`, I wrote it to use async.
But, I don't use it. So, will fix it.

Comment (0sinf) [reply src/jwt/jwt.module.ts]: When I tried to use register (not async), JwtModule couldn't get environment variables.
(Error: secretOrPrivateKey must have a value)

So, I think JwtModule need to be an async.

Comment (marshallku) [reply src/jwt/jwt.module.ts]: The example code in the [nest.js document](https://docs.nestjs.com/fundamentals/custom-providers#non-service-based-providers) uses env variables without async function. Could you check it twice?

Comment (0sinf) [reply src/jwt/jwt.module.ts]: I thought that it should be async function it for using resgisterAsync.
I try it then I can use registerAsync without async function.

useFactory defined like next capture.
useFactory type in JwtModule

So, I fix it.

**Review by marshallku — APPROVED** [review]
LGTM!

**Pull request merged**